lib/src/server/interceptor.dart

```diff
@@ -0,0 +1,7 @@
+import '../shared/status.dart';
+
+import 'call.dart';
+
+abstract class Interceptor {
+  GrpcError handle(ServiceCall call);
```

Comment:
What is use case? 
What's better way to implement? It could be
* `GrpcError.custom`, check error code  and sink stream without sending the error. 
* Exception, on any exception silently sink stream